crates/bevy_input/src/gamepad.rs

```diff
@@ -16,32 +16,33 @@ impl Gamepad {
 }
 
 #[derive(Default)]
-/// Container of unique connected [`Gamepad`]s
+/// Resource that stores the set of connected [`Gamepad`]s
 ///
-/// [`Gamepad`]s are registered and deregistered in [`gamepad_connection_system`]
+/// [`Gamepad`]s are registered and deregistered in [`gamepad_connection_system`],
+/// based on the [`GamepadEvent`]s provided by the system.
 pub struct Gamepads {
     gamepads: HashSet<Gamepad>,
```

Comment:
I deliberately decided to avoid making this field public for now: the provided methods are more ergonomic and adequately powerful.

This also lets us optimize this to some flavor of small set in the future without breaking this bit of public API.